test/test_torch.py

```diff
@@ -4187,8 +4188,7 @@ def check2d(x, y):
 
     def test_dlpack_conversion(self):
```

Comment:
I am happy to move this out to test_dlpack.py something if we want

Reply:
No, it's such a small feature that it's better to keep it here